app/src/main/java/com/example/albumsapp/network/AlbumApiService.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.albumsapp.network
+
+import com.example.albumsapp.model.Album
+import com.example.albumsapp.model.Photo
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+private const val BASE_URL = "https://jsonplaceholder.typicode.com"
+
+val moshi = Moshi.Builder()
+    .add(KotlinJsonAdapterFactory())
+    .build()
+
+val retrofit = Retrofit.Builder()
+    .addConverterFactory(MoshiConverterFactory.create(moshi))
+    .baseUrl(BASE_URL)
+    .build()
+
+
+interface AlbumApiService {
+    @GET("Albums")
+     fun getAlbums() : List<Album>
```

Comment:
I think we need to make this and `getPhotos`  functions `suspend` and later run them in background thread.
All long running operations (including network requests) have to be run in background on Android (and iOS too) because it may take a lot of time (if user internet is poor) and the app will freeze.